MonoGame.Framework/Graphics/SpriteFont.cs

```diff
@@ -50,7 +52,9 @@ public int GetHashCode(char b)
 			Spacing = spacing;
 			DefaultCharacter = defaultCharacter;
 
-			_glyphs = new Dictionary<char, Glyph>(characters.Count, CharComparer.Default);
+            _glyphsDictionary = new Dictionary<char, Glyph>(characters.Count, CharComparer.Default);
+            _glyphs = new Glyph[characters.Count];
+            var regions = new Stack<CharacterRegion>();
 
 			for (var i = 0; i < characters.Count; i++) 

```

Comment:
This should have been all done in the content pipleline and not runtime.  We should eventually research if we can increment the version on `SpriteFont` files and pack this data better for runtime use.